tests/integration/components/bs-datetimepicker-test.js

```diff
@@ -54,4 +54,28 @@ module('Integration | Component | bs datetimepicker', function(hooks) {
     await focus('input');
     assert.dom('.bootstrap-datetimepicker-widget').exists('picker should be shown');
   });
+
+  test('disabled dates are disabled', async function(assert) {
+    assert.expect(4);
+
+    this.set('disabledDates',['2016-01-02','2016-01-03']);
+    await render(hbs`{{bs-datetimepicker date='2016-01-01' disabledDates=disabledDates openOnFocus=true}}`);
+    assert.dom('input').hasValue('01/01/2016 12:00 AM');
+
+    await focus('input');
+    assert.dom("td[data-day='01/02/2016']").hasClass('disabled', 'has disabled class');
+    assert.dom("td[data-day='01/03/2016']").hasClass('disabled', 'has disabled class');
+
+    //updateDisabledDates
+    await blur('input');
+    //datetimepicker only updates if disabledDates array is already Date objects
+    let date = new Date();
+    this.set('disabledDates',[
+      new Date(1451692800000 + (date.getTimezoneOffset() * 60000)),
```

Comment:
How about just `new Date('1/1/2016')` instead?

Reply:
The trouble I ran into with that is because my timezone is negative the date selected on the calendar would be the day prior.

Reply:
Then how about `new Date('1/1/2016, 6:00:00 PM')`?

Reply:
actually your initial idea was correct, using m/d/y format is local time where yyyy-m-d is treated as utc

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#Differences_in_assumed_time_zone